src/SimpleEventStore/SimpleEventStore.AzureDocumentDb.Tests/AzureDocumentDBEventStoreInitializing.cs

```diff
@@ -150,31 +124,21 @@ public async Task set_database_and_collection_throughput_when_database_has_alrea
             await CreateDatabase(existingDbThroughput);
             var collectionName = "CollectionThroughput_" + Guid.NewGuid();
 
-            var storageEngine = await StorageEngineFactory.Create(DatabaseName,
-                collectionOptions =>
-                {
-                    collectionOptions.CollectionName = collectionName;
-                    collectionOptions.CollectionRequestUnits = collectionThroughput;
-                });
-
-            await storageEngine.Initialise();
+            var storageEngine = await InitialiseStorageEngine(collectionName, collectionThroughput, null);
```

Comment:
Third argument not required since you're already passing the default

Reply:
This test explicitly requires that `databaseThroughput` is set to null, rather than not caring what the value is, so I prefer to explicitly set the null here, rather than relying on the default.